root/static/scripts/relationship-editor/utility/compareRelationships.js

```diff
@@ -318,6 +318,28 @@ export default function compareRelationships(
     if (seriesItemCmp) {
       return seriesItemCmp;
     }
+  } else if (
+    targetA === targetB &&
```

Comment:
Maybe this is the issue, because the objects differ? (Untested.)
```suggestion
    targetIdCmp === 0 &&
```

Reply:
Hmm. The issue is what you suggested (it came up because two episodes of a podcast were released only bundled on one release, so the release needed to be linked as two numbers to the podcast series) and it actually did work when I was testing :) But it might be that the objects are not always the same? Maybe if one is not a full result of a search but saved as a recent item? I'll test.

Reply:
Ok, it wasn't working now. I really dunno why it worked on my previous test then, but in any case your suggestion works and seems sensible, so pushed it.